documents/src/pages/elements/datetime-picker.md

```diff
@@ -66,51 +66,7 @@ datetimePicker.value = '2019-03-20';
 
```

Comment:
To keep moving, please create a separate PR with documentation changes, and leave this one just for tests and comments.